certification/internal/shell/has_prohibited_packages.go

```diff
@@ -1,65 +1,28 @@
 package shell
 
 import (
-	"bufio"
-	"strings"
-
 	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
-	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
 	log "github.com/sirupsen/logrus"
 )
 
 type HasNoProhibitedPackagesCheck struct{}
 
 func (p *HasNoProhibitedPackagesCheck) Validate(image string) (bool, error) {
-	line, err := p.getDataToValidate(image)
+	result, err := podmanEngine.Unshare("HasNoProhibitedPackagesMounted", image, "./preflight", "check", "run")
 	if err != nil {
-		log.Error("unable to get a list of all packages in the image")
+		log.Error("unable to execute preflight in the unshare env")
```

Comment:
On one hand, we _need_ to log this. On the other, your average user seeing this would likely have no idea exactly how to "fix this check" without the knowledge that we've recursed (i.e. implementation details of preflight as a tool).

Maybe we need to do this in at a higher verbosity, and log simply that this check has not passed at a lower verbosity?

Reply:
Lowering to trace